lang/funcs/collection_test.go

```diff
@@ -2336,6 +2336,20 @@ func TestSlice(t *testing.T) {
 			listWithUnknowns,
 			cty.NumberIntVal(1),
 			cty.NumberIntVal(2),
+			cty.ListVal([]cty.Value{cty.UnknownVal(cty.String)}),
+			false,
+		},
+		{ // unknowns in the list
```

Comment:
The comment on this one is the same as the fixture above it -- could it be more useful? (the difference here is that this one has two listWithUnknowns?)

Reply:
thanks @pselle, updated the commit